Tests/UnitTests/Tabs/ProfileCoordinatorTests.swift

```diff
@@ -44,4 +65,38 @@ final class ProfileCoordinatorTests: XCTestCase {
         sut.updateToken(tokenHolder)
         XCTAssertEqual(try vc.emailLabel.text, "You’re signed in as\nmock@email.com")
     }
+
+    func test_openSignOutPage() throws {
+        sut.start()
+        window.rootViewController = sut.root
+        window.makeKeyAndVisible()
+        sut.openSignOutPage()
+        let presentedVC = try XCTUnwrap(sut.root.presentedViewController as? UINavigationController)
+        XCTAssertTrue(presentedVC.topViewController is GDSInstructionsViewController)
+    }
+
+    func test_tapSignoutClearsData() throws {
+        mockAnalyticsService.hasAcceptedAnalytics = true
+        try mockUserStore.secureStoreService.saveItem(item: "accessToken", itemName: .accessToken)
+        mockDefaultStore.set(Date(), forKey: .accessTokenExpiry)
+        sut.start()
+        window.rootViewController = sut.root
+        window.makeKeyAndVisible()
+        sut.openSignOutPage()
+        let presentedVC = try XCTUnwrap(sut.root.presentedViewController as? UINavigationController)
+        XCTAssertTrue(presentedVC.topViewController is GDSInstructionsViewController)
+        let signOutButton: UIButton = try XCTUnwrap(presentedVC.topViewController!.view[child: "instructions-button"])
+        signOutButton.sendActions(for: .touchUpInside)
+        XCTAssertFalse(try XCTUnwrap(mockAnalyticsService.hasAcceptedAnalytics!))
+        XCTAssertNil(try mockUserStore.secureStoreService.readItem(itemName: .accessToken))
```

Comment:
Can you also please add

`XCTAssertNil(try mockUserStore.secureStoreService.readItem(itemName: .idToken))`